saltgui/static/scripts/output.js

```diff
@@ -110,6 +167,12 @@ class Output {
         return false;
       }
 
+      // arrays are also objects,
+      // but not what we are looking for
+      if(Array.isArray(output)) {
+        return false;
+      }
+
```

Comment:
This code is now duplicated: line 164 and line 170

Reply:
That because I do not trust JS. Better do it twice...

Reply:
just kidding, it was duplicated due to cherry-pick. then after the merge we had both. cleaned up.